site/pt/blog/new-in-devtools-96/index.md

```diff
@@ -29,166 +29,195 @@ draft: true
 {% include 'partials/devtools/pt/banner.md' %}
 
 <!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
-
+## Prévia de funcionalidade: novo painel de visão geral CSS  {: #css-overview }
 <!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
 [Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
 
+Use o novo painel **Visão geral CSS** para identificar possíveis melhorias de CSS em sua página.
+[Abra o painel **Visão geral CSS**](/docs/devtools/css-overview#open) e clique em **Capture visão geral** para gerar um relatório do CSS da sua página.
+
 <!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+Você pode detalhar ainda mais as informações. Por exemplo, clique em uma cor na seção **Cores** para visualizar a lista de elementos que aplicam a mesma cor. Clique em um elemento para abri-lo no painel **Elementos**.
```

Comment:
Wondering if "expandir" reads better as a translation for drill down than "detalhar".
```suggestion
Você pode expandir as informações. Por exemplo, clique em uma cor na seção **Cores** para visualizar a lista de elementos que aplicam a mesma cor. Clique em um elemento para abri-lo no painel **Elementos**.
```